app/build.gradle

```diff
@@ -135,6 +137,9 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
```

Comment:
This should probably be `testImplementation` and also further down with the test dependencies.

Reply:
Good catch, fixed it